README.md

```diff
@@ -34,7 +34,7 @@ read the [code of conduct](https://github.com/devconzm/devconzm/blob/master/CODE
 - git
 
 clone the repo
```

Comment:
add double space at the end of repo so that both lines are separated

Reply:
alright